diff --git a/matlab/private/invoke_openEMS.m b/matlab/private/invoke_openEMS.m
index ce3ac53..d64771d 100644
--- a/matlab/private/invoke_openEMS.m
+++ b/matlab/private/invoke_openEMS.m
@@ -32,7 +32,9 @@ end
 filename = mfilename('fullpath');
 dir = fileparts( filename );
 openEMS_Path = [dir filesep '../..' filesep];
-    
+
+openEMS_Path = ''; % this is a packaged openEMS installation; openEMS.sh is found in $PATH
+
 if isunix
 	openEMS_Path = [openEMS_Path 'openEMS.sh'];
 else
